### PR TITLE
Fix `.alert-success` System Test Error

### PR DESCRIPTION
**Pull request by noi5e**
Fix for #10140

### CODE REVIEW

File: test/system/post_test.rb [hunk 1]
```diff
@@ -247,12 +247,9 @@ def assert_page_does_not_reload(message = "page should not reload")
     find("span[data-original-title='Tools']").click()
     find("input[value='Give']").click()
 
-    # Wait for the alert to be shown
-    wait_for_ajax
```
Comment (noi5e) [inline test/system/post_test.rb:251]: Removing `wait_for_ajax` here, I think in my prior days of writing system tests, `page.find` implicitly waited for AJAX updates.

File: test/system/post_test.rb [hunk 2]
```diff
@@ -247,12 +247,9 @@ def assert_page_does_not_reload(message = "page should not reload")
...
-
-    page.assert_selector("div.alert-success", text: "You awarded the basic barnstar to #{note.author.name}")
-    page.assert_selector("p", text: "#{note.author.name} was awarded the Basic Barnstar by palpatine for their work in this research note.")
-    page.assert_selector(".comment-body p", text: "@palpatine awards a barnstar to #{note.author.name} for their awesome contribution!")
+    page.find("div.alert-success", text: "You awarded the basic barnstar to #{note.author.name}")
+    assert_selector("p", text: "#{note.author.name} was awarded the Basic Barnstar by palpatine for their work in this research note.")
+    assert_selector(".comment-body p", text: "@palpatine awards a barnstar to #{note.author.name} for their awesome contribution!")
```
Comment (noi5e) [inline test/system/post_test.rb:252]: I think `assert_selector` is enough here, no need to `page.assert_selector`.

**Comment by noi5e** [comment]
@jywarren Tests are passing here!

I suggest we merge this and see if it fixes the issue over the long run?

**Comment by jywarren** [comment]
Excellent, thank you @noi5e !!!

**Pull request merged**